data/json/items/comestibles/alcohol.json

```diff
@@ -1425,6 +1425,34 @@
     "fun": 25,
     "vitamins": [ [ "vitA", 2 ], [ "vitC", 23 ], [ "calcium", 1 ], [ "iron", 2 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "drink_bloody_mary_blood",
+    "name": { "str_sp": "'exalted' bloody mary cocktail" },
```

Comment:
```suggestion
    "name": { "str_sp": "'genuine' bloody mary cocktail" },
```

Reply:
"genuine", "classic", "original", "simple", "true" or "real" just not working here.
alternative candidate:
"special": stale and boring but OK
"exalted", "glorious", "your excellency": What I'm using now, for historical Bloody Mary(1516-1558)
"naughty", "bitchy", "puckish", "~~milf~~": What I want exactly but somehow inappropriate..?

Reply:
The references that mention "exalted", "glorious", "your excellency" refer to bloody mary, the person, as far as I can tell. I don't know if it makes sense to refer to the drink as one would a noble, even if it was created in reference to queen Mary. For that matter, I don't think it would make sense to refer to a drink as a person, unless it is actually made with human blood.

Reply:
'hysterical' sounds appropriate. But I just want to make sure it's OK to put this word on Bloody "Mary"(A girl name)
Well you know, to "avoid direct association with gender", I have to do censorship both here and in my brain🤕.